runtime/rasdump/dmpagent.c

```diff
@@ -886,7 +886,16 @@ static omr_error_t
 doJavaDump(J9RASdumpAgent *agent, char *label, J9RASdumpContext *context)
 {
 	J9JavaVM *vm = context->javaVM;
-	if ((0 != strcmp(label, "/STDOUT/")) && (0 != strcmp(label, "/STDERR/"))) {
+	
+	if ('-' == label[0] || 0 == strcasecmp(label, J9RAS_STDOUT_NAME)) {
+                label = J9RAS_STDOUT_NAME;
+        } 
+        
+	if (0 == strcasecmp(label, J9RAS_STDERR_NAME)) {
+                label = J9RAS_STDERR_NAME;
+        }
+	
+	if ((0 != strcmp(label, J9RAS_STDOUT_NAME)) && (0 != strcmp(label, J9RAS_STDERR_NAME))) {
```

Comment:
Please fix the indentation (use leading tabs).
Please add parentheses around the parts of compound conditions.
Use `if` / `else if` to avoid repeating (the incorrect) checks for stdout/stderr.
This function is expected to write into the buffer addressed by `label` the name of the output file; use `strcpy` for the stdout and stderr cases.

Reply:
Sloppy coding on my end, that was my fault. I'll run a personal build with my changes before pushing the changes.

Reply:
If you were talking about the repetition on line 898, we need to have a separate `if` statement since the first 2 if statements will change `-`, `/stdout/` and `/stderr/` into `/STDOUT/` and `/STDERR/` respectively.